### PR TITLE
Increasing coverage

### PR DESCRIPTION
**Pull request by nadiashovkovy**
I tried to increase the coverage.

### CODE REVIEW

File: datatableview_user_columns/tests/test_datatables.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        self.assertEqual(sls.table_name, "test_table")
+        self.assertEqual(user.username, "Nadia")
+        print(sls)
+
+    def test_column_edit_url(self):
+        user = User.objects.create_user("Nadia")
+        sls2 = DataTableUserDataTableMixin(table_name="test_table", user=user)
+        self.assertEqual(sls2.column_edit_url, '/user_columns/create/test_table/')
+
+    def test_column_delete_url(self):
```
Comment (nadiashovkovy) [inline datatableview_user_columns/tests/test_datatables.py]: hi

File: .coverage
Comment (BradyD11) [inline .coverage:1]: delete this

**Review by BradyD11 — COMMENTED** [review]
From the standpoint of solely analyzing file changes, deletions and additions I had no issues that were major enough or that I noticed

File: .coverage
Comment (BradyD11) [reply .coverage:1]: except this one

**Pull request merged**